### ISSUE TITLE
Possible to add ability to ask Alexa what's playing?

### ISSUE THREAD

**Issue by sacarino** [julianh2o/RokuAlexaLambdaSkill#26]
This looks pretty cool - going to carve some time over the holiday to set it up and see if my wife will accept it or not. :)

Similar to how you can ask Alexa to identify songs it's currently playing, it'd be pretty awesome if you could ask it what's currently being played. Is that possible?

I didn't see anything in the ECP that jumps out as a `yes`, so I started looking into the other SDK docs. Roku's documentation is... well, a bit of a mess.

**Comment by mikehobi** [comment]
@sacarino I don't believe this is possible. It seems that the only information you can query for is the current active channel (eg. Netflix, Amazon).

Could be something in the future if Roku updates their SDK though

**Comment by julianh2o** [comment]
Yeah, unfortunately the Roku ECP is almost completely devoid of endpoints
that provide information about the state of the device. That's why most of
this code is based on keypress sequences that happen to play the right
thing. It'd be great to even have a "finished loading" status indicator..
but no sign of that.

**Comment by sacarino** [comment]
That's what I thought as well, but I was hoping that I had overlooked something non-intuitive like a BrightScript payload. Seems crazy to have a device they want to be the center of everything, yet doesn't provide very many useful outputs. 😐 

Oh well, guess this goes in the icebox!
